.editorconfig

```diff
@@ -1,14 +0,0 @@
-root = true
```

Comment:
Are you sure this works to use a filepath, for PHPStorm etc. ? 🤔

I would not touch the files from dev kit itself in this PR and would Not couple the project files with the dev kit app anyway. In this case I would prefer duplication

Reply:
Do you want duplication for all of config file (phpcs, yamllint, flintci, ...) or just for editorconfig ?

Reply:
I just want to keep it untouched we’re we did t use paths like now. Anyway are you sure this works for php cs fixer? 🤔

Reply:
It work like this for editorconfig.

It wasn't for phpcs. I made a fix.